api/proto/common/common.proto

```diff
@@ -3,8 +3,47 @@ package common;
 option go_package = "github.com/Layr-Labs/eigenda/api/grpc/common";
 
 message G1Commitment {
-	// The X coordinate of the KZG commitment. This is the raw byte representation of the field element.
-	bytes x = 1;
-	// The Y coordinate of the KZG commitment. This is the raw byte representation of the field element.
-	bytes y = 2;
+  // The X coordinate of the KZG commitment. This is the raw byte representation of the field element.
+  bytes x = 1;
+  // The Y coordinate of the KZG commitment. This is the raw byte representation of the field element.
+  bytes y = 2;
 }
+
+/////////////////////////////////////////////////////////////////////////////////////////////////
+// Messages for WIP RPCs. These are subject to change, do not consider these to be stable API. //
+/////////////////////////////////////////////////////////////////////////////////////////////////
+
+// A unique identifier for a blob.
+message BlobKey {
+  oneof key {
```

Comment:
Do we have to support both? It makes the golang code more verbose and less easy to work with (have to check which field is set each time)

Reply:
This is probably a good discussion topic with @mooselumph, the plan for how to address a blob is something he's been working on.